src/object.js

```diff
@@ -37,5 +37,62 @@ _.extend(Marionette.Object.prototype, Backbone.Events, {
   bindEntityEvents: Marionette.proxyBindEntityEvents,
 
   // Proxy `unbindEntityEvents` to enable unbinding view's events from another entity.
-  unbindEntityEvents: Marionette.proxyUnbindEntityEvents
+  unbindEntityEvents: Marionette.proxyUnbindEntityEvents,
```

Comment:
I don't want to add too much more work for you, but I think this deserves it's own file.

```
src/proxyRadioHandlers.js
```